wpilibj/src/athena/java/edu/wpi/first/wpilibj/ADXL362.java

```diff
@@ -49,7 +49,7 @@
     @SuppressWarnings("MemberName")
     public final byte value;
 
-    private Axes(byte value) {
+    Axes(byte value) {
```

Comment:
Ah ok. So these are ok. But the ones turning public to package private still are changing, right?